playbooks/byo/openshift-preflight/lookup_plugins

```diff
@@ -0,0 +1 @@
+../../../lookup_plugins
```

Comment:
> We do not directly.​ At some point in development I ran into a role that
used them, took a while to figure out what was going on, and I didn't want
to have to do that again. Could be nothing in this needs them now however
it's also future-proofing in case dependent roles start using them.

Hmm, if we don't understand well the problem, I'm afraid adding it could have the opposite effect, plus bring in perhaps unintended side-effects (like Ansible doing things we do not need nor anticipate).

Very likely whenever we run the checks Ansible would unnecessarily look at those dirs and load the plugins? I remember I couple of times I ran `strace ... ansible-playbook ...` to track down where ansible was looking for files.

I'd prefer to avoid the bloat over questionable "future-proofing" -- does such a thing even exist in practice? :D
I still remember when I read Ron Jeffries' book! https://en.wikipedia.org/wiki/YAGNI